src/MainApp.js

```diff
@@ -0,0 +1,28 @@
+import React, { useContext } from "react";
+import Main from "./components/Main";
+import Sidebar from "./components/Sidebar";
+import Navi from "./components/Navi";
+import NoteDisplay from "./components/NoteDisplay/";
+import { ThemeContext } from "./contexts/ThemeContext";
+import NoteInfo from "./components/NoteInfo";
+import { AppContext } from "./contexts/AppContext";
+
+export default function MainApp() {
+    const { darkMode } = useContext(ThemeContext);
+    const { state } = useContext(AppContext);
+
+    return (
+        <React.Fragment>
```

Comment:
What do you think about using the shorter hand for `React.Fragment` like this `<> </>`? Since it is like a "ghost" div but it's not really included the compiled result.